src/graphql/admin/root/mutation/account-update-level.ts

```diff
@@ -17,7 +17,16 @@ const AccountUpdateLevelInput = new GT.Input({
   }),
 })
 
-const AccountUpdateLevelMutation = GT.Field({
+const AccountUpdateLevelMutation = GT.Field<
+  null,
+  GraphQLContext,
+  {
```

Comment:
the third object is the type of input?

Reply:
Yep. It's the type of the `args` object.

Reply:
Ok, I made args the first argument since the others can be set default.

So the order now is TArgs, TSource, TContext with the TSource defaulting to null and TContext defaulting to GraphQLContext

We have a ton of errors already. Will work through them soon.